src/components/lyne-toast/styles/lyne-toast.component.scss

```diff
@@ -0,0 +1,113 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin, available for shared
+// and default mode
+@include host-component-properties;
+
+:host {
+  --color-default-text: var(--color-white-default);
+  --toast-background-color: var(--color-midnight-default);
+  --toast-border-radius: #{pxToRem(var(--border-radius-4x))};
+  --toast-gap: #{pxToRem(var(--spacing-fixed-4x))};
+  --toast-padding-block: #{pxToRem(var(--spacing-responsive-xxxs))};
+  --toast-padding-inline: #{pxToRem(var(--spacing-responsive-xs))};
+  --toast-margin: #{pxToRem(var(--spacing-responsive-s))};
+  --toast-max-width: #{pxToRem(400)};
+  --toast-width: #{calc(100% - (2 * pxToRem(var(--spacing-responsive-s-zero))))};
+}
+
+.overlay-hidden {
+  display: none;
+}
+
+.toast-wrapper {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  contain: content;
+}
+
+.toast {
+  @include text-s--bold;
+  @include shadow-level-5-hard;
+  @include absoluteCenterX;
+
+  position: fixed;
+  padding-block: var(--toast-padding-block);
+  padding-inline: var(--toast-padding-inline);
+  margin: var(--toast-margin) 0;
+  width: calc(100% - (2 * pxToRem(var(--spacing-responsive-s-zero))));
+
+  @include mq($from: small) {
+    width: var(--toast-max-width);
+  }
+
+  display: flex;
+  flex-direction: row;
+  align-items: start;
+  bottom: 0;
+  gap: var(--toast-gap);
+  background: var(--toast-background-color);
+  color: var(--color-default-text);
+  border-radius: var(--toast-border-radius);
+
+  &.toast-top {
+    top: 0;
+    bottom: auto;
+  }
+
+  &.toast-left {
+    left: var(--toast-margin);
+    transform: none;
+  }
+
+  &.toast-right {
+    left: auto;
+    right: var(--toast-margin);
+    transform: none;
+  }
+}
+
+.toast-text {
+  flex-grow: 1;
+}
+
+// FIXME temporary, waiting for sbb-icon
+.toast-icon {
```

Comment:
as mentioned in call: spacing not correct and value should probably be 24, respectively its css var.